packages/accounts-controller/src/index.ts

```diff
@@ -1,2 +1,18 @@
-export * from './AccountsController';
-export * from './utils';
+export type {
+  AccountsControllerState,
+  AccountsControllerGetStateAction,
+  AccountsControllerSetSelectedAccountAction,
+  AccountsControllerSetAccountNameAction,
+  AccountsControllerListAccountsAction,
+  AccountsControllerUpdateAccountsAction,
+  AccountsControllerGetSelectedAccountAction,
+  AccountsControllerGetAccountByAddressAction,
+  AccountsControllerGetAccountAction,
+  AccountsControllerActions,
+  AccountsControllerChangeEvent,
+  AccountsControllerSelectedAccountChangeEvent,
+  AccountsControllerEvents,
+  AccountsControllerMessenger,
```

Comment:
- This is not a breaking change. The list includes all preexisting exports and excludes `Allowed{Actions,Events}`.
- `Allowed{Actions,Events}` should be exported from the module so they're available for tests and other internal files, but they should not be exported at the package level.
- We want to avoid wildcard exports in general: https://github.com/MetaMask/eslint-config/issues/331